build.sbt

```diff
@@ -1,39 +1,36 @@
-val scala2_11 = "2.11.12"
 val scala2_12 = "2.12.8"
-lazy val commonSettings = commonSmlBuildSettings ++ ossPublishSettings ++ Seq(
-  organization := "com.softwaremill.tapir",
-  scalaVersion := scala2_12,
-  scalafmtOnCompile := true,
-  crossScalaVersions := Seq(scala2_11, scala2_12),
-  scalacOptions ++=
-    (CrossVersion.partialVersion(scalaVersion.value) match {
-      case Some((2, 11)) =>
-        Seq(
-          "-Xexperimental",
-          "-Ypartial-unification"
-        )
-      case _ =>
-        Nil
-    })
-)
+val scala2_13 = "2.13.0"
+def commonSettings(scalaVersions: Seq[String] = Seq(scala2_12, scala2_13)) =
+  commonSmlBuildSettings ++ ossPublishSettings ++ Seq(
+    organization := "com.softwaremill.tapir",
+    scalaVersion := scala2_13,
```

Comment:
shouldn't this use the first (?) scala version from `scalaVersions`?

Reply:
IDK, can do that.